platform/android/app/build.gradle

```diff
@@ -1,16 +1,19 @@
+import java.text.SimpleDateFormat
```

Comment:
Is this something that became available in the new versions you're using?

Reply:
We used this because Date.parse was removed.